components/codec/convert.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const SIGN_MARK: u64 = 0x8000000000000000;
```

Comment:
Why not `1 << 63` ? It's less likely to make mistake about the number of zeros

Reply:
It is copied from current code. I didn't have a strong opinion so I didn't changed it. But your worry is reasonable. I will change it.

Reply:
Changed, PTAL, thanks!